### DISCUSSION TITLE
PostgreSQL의 장점은 무엇일까요?

### DISCUSSION THREAD

**Discussion by eatingbug** [Team-Psyduck/tech-sharing#67] [General]
-

**Comment by eatingbug** [comment]
PostgreSQL 의 장점은 대표적으로
1. 관계형 DB 중에서 **가장 많은 SQL과 표준을 지원**한다.
2. 일부 NoSQL도 지원하는 등 **풍부한 데이터 유형 지원**
3. **다양한 프로그래밍 언어 지원**
4. `Table Partitioning`, `Parallel query & multiple processes` 등의 기능을 지원하여 **대량 데이터 처리가 가능**하다.

**Comment by SionBackEnd** [comment]
PostgreSQL의 장점
1. 라이센스 비용이 없어서 무료로 이용이 가능하다. 
2. 가장 높은 수준으로 SQL 표준을 준수한다. (다른 데이터 베이스로 이전이 수월함)
3. Partial index(조건별로 필터링한 index)와 더욱 빠른 인덱스 생성을 위한 병렬 처리를 지원한다.
4. NoSQL지원(key-value)한다.
5. JSON 인덱싱을 지원한다.
[참고](https://dataleader.tistory.com/34)
